TourCMS/TourCMS.cs

```diff
@@ -64,7 +63,9 @@ public XmlDocument Request(string path, int channelId)
         /// </summary>
         public XmlDocument Request(string path, int channelId, string verb)
         {
-            return Request(path, channelId, "GET", null);
+            if (string.IsNullOrEmpty(verb))

```

Comment:
I don't think we need this check here.

We have several overrides, so the caller should be using the correct override (the one with only 2 params if they aren't passing a verb). Any check/defaulting we should just do in the main "Request" function.